### PR TITLE
fix lab scope

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: tests/Microsoft.Identity.Test.LabInfrastructure/LabApiConstants.cs [hunk 1]
```diff
@@ -37,10 +37,7 @@ public class LabApiConstants
         public const string True = "true";
         public const string False = "false";
 
-        public const string BetaEndpoint = "http://api.msidlab.com/api/userbeta";
-        //public const string LabEndpoint = "http://api.msidlab.com/api/user";
-        public const string CreateLabUser = "https://request.msidlab.com/api/CreateLabUser";
-        public const string LabEndPoint = "https://user.msidlab.com/api/user";
+        public const string LabEndPoint = "https://msidlab.com/api/user";
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.LabInfrastructure/LabApiConstants.cs]: @jennyf19 - this also "needed" to change.

File: tests/Microsoft.Identity.Test.LabInfrastructure/LabServiceApi.cs [hunk 2]
```diff
@@ -151,18 +151,6 @@ private async Task<string> GetLabResponseAsync(string address)
             }
         }
 
-        public async Task<LabResponse> CreateTempLabUserAsync()
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.LabInfrastructure/LabServiceApi.cs]: Remove unused functionality

File: tests/Microsoft.Identity.Test.LabInfrastructure/LabApiConstants.cs (hunk 1 above)
Comment (jennyf19) [reply tests/Microsoft.Identity.Test.LabInfrastructure/LabApiConstants.cs]: i thought i had changed that. i did in the adal one.

File: tests/Microsoft.Identity.Test.LabInfrastructure/LabServiceApi.cs [hunk 3]
```diff
@@ -56,12 +56,6 @@ private async Task<LabResponse> GetLabResponseFromApiAsync(UserQuery query)
         private async Task<LabResponse> CreateLabResponseFromResultStringAsync(string result)
         {
             LabUser[] userResponses = JsonConvert.DeserializeObject<LabUser[]>(result);
-            if (userResponses.Length > 1)
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.LabInfrastructure/LabServiceApi.cs]: Hmm, does it return multiple users now?

**Comment by bgavrilMS** [comment]
This works, but there are MSA tests that are failing because the password was changed. Orthogonal issue. Committing the change.

**Pull request merged**